src/EventManagement.Services/IRegistrationService.cs

```diff
@@ -23,6 +23,7 @@ public interface IRegistrationService
 
 		Task<int> CreateRegistration(Registration registration);
 		Task<int> CreateRegistration(Registration registration, int[] productIds, int[] variantIds);
+		Task<bool> AddRegistrationToProduct(string email, int eventId, int productId, int? variantId);
```

Comment:
Should this be part of ordersservice instead? And be renamed to AddOrderForProduct? as well as the method below.

Reply:
I don't think so. Since it's basically an operation on the registration.
The method could however be renamed to `AddProductToRegistration`.

What do you think?